packages/server/database/migrations/20221212161224-updateTier.ts

```diff
@@ -0,0 +1,181 @@
+import {R} from 'rethinkdb-ts'
+import getPg, {closePg} from '../../postgres/getPg'
+
+export const up = async function (r: R) {
```

Comment:
looking now...

Reply:
migration is borking here: https://github.com/ParabolInc/parabol/actions/runs/3675781184/jobs/6215620473#step:5:229
because we already dropped the Team table here: https://github.com/ParabolInc/parabol/blob/234bc53534684b29b70e2a4357ba27f42bd1fa20/packages/server/database/migrations/20220329101759-dropTeams.ts#L22

Reply:
Thanks! I removed the Team bit of the migration, but I'm still getting the same [typecheck error](https://github.com/ParabolInc/parabol/actions/runs/3733494175/jobs/6334406317).

In `addPokerTemplate`, for example, the typecheck error complains that [viewerTeam.tier](https://github.com/ParabolInc/parabol/blob/2b8bb25f1df75c0b228a9230026ca3f9a9088c85/packages/server/graphql/mutations/addPokerTemplate.ts#L58) can't be `starter`, but `viewerTeam` is of type [Team](https://github.com/ParabolInc/parabol/blob/2114d28ba7315e75f2dae1bc958edea15b3f40cc/packages/server/database/types/Team.ts#L14), which gets `TierEnum` from Invoice [here](https://github.com/ParabolInc/parabol/blob/2b8bb25f1df75c0b228a9230026ca3f9a9088c85/packages/server/database/types/Invoice.ts#L7).

Locally, when I run `yarn typecheck`, everything passes, but it continues to fail here 🤔

Reply:
Moving the pg bit of the migration to a separate migration fixed the typecheck error. Thanks @Dschoordsch 🙏